### PR TITLE
Removed Warnings

### LINKED ISSUE #170: Remove Warnings
- [x] I am running the latest version
- [x] I checked to make sure that this issue has not already been filed

# Expected Behavior
        Project Raise no warning 

# Current Behavior
What is the current behavior?
         Raising Warnings For Different Purposes

### PR DESCRIPTION
**Pull request by shakti97**
closes #170

### CODE REVIEW

File: fact-bounty-client/src/pages/login/components/Login.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import FormControl from '@material-ui/core/FormControl';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import withStyles from '@material-ui/core/styles/withStyles';
+
```
Comment (Tushar19varshney) [inline fact-bounty-client/src/pages/login/components/Login.js]: Double quotes are used throughout the client side.
So changing it to single is unnecessary!

File: fact-bounty-client/src/pages/register/components/Register.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-import PropTypes from "prop-types";
-import { connect } from "react-redux";
-import classnames from "classnames";
-import { Link as RouterLink } from "react-router-dom";
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import classnames from 'classnames'
+
```
Comment (Tushar19varshney) [inline fact-bounty-client/src/pages/register/components/Register.js]: Again Quotes issue!

**Comment by Tushar19varshney** [comment]
Use Codacy standards for formatting the code and have a look at the Codacy test that failed.

**Comment by root5533** [comment]
Squash commits and please check Codacy and fix your code.

**Comment by root5533** [comment]
Feature is not required

**Pull request closed without merging**